### ISSUE TITLE
airbrake-rails breaks when ActiveRecord is not available

### ISSUE THREAD

**Issue by vihai** [airbrake/airbrake#925]
Hello,

I have a small frontend application for which I don't load ActiveRecord, howerver a recent update triggers the following error:

```
NameError: uninitialized constant ActiveRecord
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/active_support.rb:60:in `block in load_missing_constant'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/active_support.rb:16:in `allow_bootsnap_retry'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/active_support.rb:59:in `load_missing_constant'
/var/lib/gems/2.5.0/gems/airbrake-8.2.1/lib/airbrake/rails/active_record_subscriber.rb:42:in `<main>'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/var/lib/gems/2.5.0/gems/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
/var/lib/gems/2.5.0/gems/airbrake-8.2.1/lib/airbrake/rails.rb:10:in `block in <class:Railtie>'
```

**Comment by rwojnarowski** [comment]
It's fixed but not released yet https://github.com/airbrake/airbrake/pull/924

**Comment by kyrylo** [comment]
Duplicate of https://github.com/airbrake/airbrake/issues/922.
I will release today.

**Comment by vihai** [comment]
Thank you!

**Comment by arekt** [comment]
I would advice to keep issue open, until new version of the gem is released, using labels might help as well, like "ready for release" :)

**Comment by kyrylo** [comment]
We close issues when they're fixed on master. If we keep them open until a release, it's likely we'll forget to close them.

Either way, I just released v8.3.0 with the fix.
